diff --git a/node_modules/@pythnetwork/solana-utils/dist/esm/jito.mjs b/node_modules/@pythnetwork/solana-utils/dist/esm/jito.mjs
index 76eebc1..43df929 100644
--- a/node_modules/@pythnetwork/solana-utils/dist/esm/jito.mjs
+++ b/node_modules/@pythnetwork/solana-utils/dist/esm/jito.mjs
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */ /* eslint-disable @typescript-eslint/no-non-null-assertion */ /* eslint-disable @typescript-eslint/prefer-nullish-coalescing */ import { PublicKey, SystemProgram } from "@solana/web3.js";
 import bs58 from "bs58";
-import { Bundle } from "jito-ts/dist/sdk/block-engine/types";
+import { Bundle } from "jito-ts/dist/sdk/block-engine/types.js";
 import { dummyLogger } from "ts-log";
 export const TIP_ACCOUNTS = [
